testing/TransactionLogFiles.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
```

Comment:
see #58 that tries to move tests to what I have found to be more pythonic.
It uses the `unittest` module to run and give feadback if any issue are found.